The GNU Libc Reference manual has to be removed for licensing reasons.
But some files have a dependency on manual/errno.texi; the easiest
solution is to drop those dependencies and make sure that MAKEINFO=:
so that no target depends on manual/*.texi files.

---
 stdio-common/Makefile      | 2 +-
 sysdeps/mach/hurd/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/stdio-common/Makefile
+++ b/stdio-common/Makefile
@@ -92,7 +92,7 @@
 	     tst-printf-fp-leak.mtrace tst-printf-fp-leak-mem.out
 endif
 
-tests-special += $(objpfx)tst-errno-manual.out
+# tests-special += $(objpfx)tst-errno-manual.out
 
 include ../Rules
 
--- a/sysdeps/mach/hurd/Makefile
+++ b/sysdeps/mach/hurd/Makefile
@@ -88,7 +88,7 @@
 	    -e 's, \.\./, $(..),g' > $@t
 	mv -f $@t $@
 
-$(hurd)/bits/errno.h: $(common-objpfx)stamp-errnos ;
+$(hurd)/bits/errno-disabled.h: $(common-objpfx)stamp-errnos ;
 $(common-objpfx)stamp-errnos: $(hurd)/errnos.awk $(errno.texinfo) \
 			      $(mach-errnos-deps) $(common-objpfx)errnos.d
 	mkdir -p $(hurd-objpfx)bits
